src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -37,17 +40,10 @@
                     "  rect.right <= (window.innerWidth || document.documentElement.clientWidth));\n" +
             "} else { return null; }";
 
-    private static final String TOP_ELEMENT_AT =
```

Comment:
Did you intentionally remove the 'is on top' functionality? Or did you just base your code on an earlier version that did not have it yet, and did something go wrong in a merge?

If it is intentional please describe why. We will have to find a way to make this behavior optional, because it is very desirable in some standard browser based testing.